docs/README.md

```diff
@@ -185,11 +185,11 @@ Server:
 #### Creating a local AMP cluster
 
 ```
-$ amp cluster create
+$ amp cluster create --tag [AMP version]
 ```
->*If you are using the unstable version of the CLI, you will need to build the `amp-bootstrap`
-image using the command `make build-bootstrap` or `ampmake build-bootstrap` if your don't have the build tools,
-and then specify the `--tag=local` option to the CLI*
+>*If you are using the unstable version of the CLI, you don't need to add the `--tag`
+option as it will use the development version as the image tag, which has already been
+built with `build-cli`*
```

Comment:
I would use `development` instead of `unstable`, that would not make any negative assumption on the stability of AMP :)
Also, instead of `make build-cli`, let's talk about `ampmake build`, this way we're sure all the images are available.